demo-shell/src/app/components/app-layout/cloud/people-groups-cloud.component.html

```diff
@@ -0,0 +1,52 @@
+<mat-card class="adf-people-groups-section">
+    <mat-card-title>
+        {{ 'APP_LAYOUT.PEOPLE_CLOUD' | translate}}
+    </mat-card-title>
+
+    <mat-card-content>
+        <div class="people-control-options">
+            <mat-checkbox (change)="onChangePeopleMode($event)">{{ 'PEOPLE_GROUPS_CLOUD.MULTIPLE_VALUES' | translate }}</mat-checkbox>
```

Comment:
can we change the checkbox with radiobutton and have the vale as part of the HTML ?
```
 <mat-radio-group [(ngModel)]="peopleMode">
        <mat-radio-button class="example-margin" value="PeopleCloudComponent.MODE_SINGLE">Single</mat-radio-button>
        <mat-radio-button class="example-margin" value="PeopleCloudComponent.MODE_MULTIPLE">Multi</mat-radio-button>
      </mat-radio-group>
```